scripts/plugins/anchor-markdown-headings.js

```diff
@@ -42,8 +42,8 @@ module.exports = function anchorMarkdownHeadings (text, level, raw) {
 
   anchorTitle = anchorTitle.toLowerCase()
 
-  return '<h' + level + ' id="header-' + anchorTitle + '">' + text + '<a name="' +
-    anchorTitle + '" class="anchor" href="#' +
-    anchorTitle + '" aria-labelledby="header-' +
-    anchorTitle + '"></a></h' + level + '>'
+  const anchorId = `${slugger ? slugger.slug(anchorTitle) : anchorTitle}`
+  const headerId = `header-${anchorId}`
```

Comment:
Maybe this logic is wrong and I should use `slugger` here too. I'll try to verify again later.